dashboard/app/models/levels/level.rb

```diff
@@ -187,6 +187,10 @@ def self.flower_types
   def self.palette_categories
   end
 
+  # Overriden by different level types.
+  def self.hoc_songs
+  end
+
```

Comment:
ok to omit this method definition once you are ensuring that it is only called for Dancelab.